src/ApproxConv.jl

```diff
@@ -168,6 +174,20 @@ function calcVariableCovarianceBasic(ptsArr::Vector{Vector{Float64}})
   return msst_
 end
 
+function calcVariableCovarianceBasic(M::AbstractManifold, ptsArr::Vector{P}) where P
+  #TODO double check the maths,. it looks like its working at least for groups
+  μ = mean(M, ptsArr)
+  Xcs = vee.(Ref(M), Ref(μ), log.(Ref(M), Ref(μ), ptsArr))
+  Σ = mean(Xcs .* adjoint.(Xcs))
+  @debug "calcVariableCovarianceBasic" μ
+  @debug "calcVariableCovarianceBasic" Σ
+  # TODO don't know what to do here so keeping as before, #FIXME it will break
+  # a change between this and previous is that a full covariance matrix is returned
+  msst = Σ
```

Comment:
Should this perhaps be the trace to give a single variance:
```suggestion
  msst = tr(Σ)
```

Reply:
um, probably not that important here since the next line just checks if there are covariances that can work and then scale all according the maximum covariance.  Trace would also work and just scale all by the sum rather than the maximum

Reply:
ok